src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -303,6 +303,8 @@ private void onLoaded(object sender, RoutedEventArgs e)
                 btnAccountConfig.SetValue(AutomationProperties.NameProperty, Properties.Resources.btnConfigAutomationPropertiesNameNoBugFiling);
             }
 
+            this.btnMaxFabric.GlyphName = (this.WindowState == WindowState.Maximized) ? FabricIcon.ChromeRestore : FabricIcon.Checkbox;
+
```

Comment:
How about combining this line and other changes into a single method? 
it would let us touch one part of code in case we need to touch this max button case  again.